README.md

```diff
@@ -2,20 +2,28 @@ Visu - Front
 
 ## Install
 
-    $ npm ci
-    $ cp public/env.dist.json public/env.json
-    $ cp public/settings.dist.json public/settings.json
+    npm ci
+    cp public/env.dist.json public/env.json
+    cp public/settings.dist.json public/settings.json
 
 You may have to change some data in your personnal env.json and settings.json files.
 
 ## Start
 
-    $ npm start
+    npm start
 
 ## Build
 
-    $ npm run build
+    npm run build
 
 ## Tests
 
-    $ npm test
+    npm test
+
+## To execute e2e tests
+
+To execute locally the pipeline, you have to install the last version of [gitlab runner](https://docs.gitlab.com/runner/install/) then execute:
```

Comment:
To run the pipeline locally, you need to install the latest version of ...

Reply:
Fixed